main.py

```diff
@@ -75,6 +81,26 @@ def main():
     except Exception as e:
         logger.error("Error: %s", e, exc_info=True)
 
+def convert_notebooks(config_loader: ConfigLoader, notebook_paths: Optional[List[str]] = None) -> None:
```

Comment:
Сейчас вместо Optional и Union (python 3.10+) можно так делать, довольно удобно и код чище:
Union[str, int] = str | int
Optional[str] = str | None

```suggestion
def convert_notebooks(config_loader: ConfigLoader, notebook_paths: List[str] | None = None) -> None:
```